dev-cpp/opentelemetry-cpp/opentelemetry-cpp-1.3.0-r2.ebuild

```diff
@@ -51,3 +51,12 @@ src_configure() {
 
 	cmake_src_configure
 }
+
+src_install() {
+	cmake_src_install
+
+	if use prometheus; then
```

Comment:
I think it's better to patch this in. It's too easy for the sed to do nothing and nobody notice.